From 346abfb3a98c0b527bb8e73ff1978dbd32684c17 Mon Sep 17 00:00:00 2001
From: babak sarashki <babak.sarashki@windriver.com>
Date: Thu, 31 Oct 2019 10:16:21 -0700
Subject: [PATCH] Set rabbitmq dependency to nanliu/staging

Stx package puppet-staging-1.0.4-1.b466d93git sets module name to:
nanliu-staging.
This patch sets rabbitmq dependency accordingly.
---
 metadata.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index b3426f6..c4297b2 100644
--- a/metadata.json
+++ b/metadata.json
@@ -48,6 +48,6 @@
   ],
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">= 3.13.1 < 5.0.0"},
-    {"name":"puppet/staging","version_requirement":">= 0.3.1 < 2.0.0"}
+    {"name":"nanliu/staging","version_requirement":">= 0.3.1 < 2.0.0"}
   ]
 }
-- 
2.17.1

